pkgs/desktops/gnome-3/extensions/no-title-bar/default.nix

```diff
@@ -2,18 +2,16 @@
 
 stdenv.mkDerivation rec {
   pname = "gnome-shell-extension-no-title-bar";
-  version = "9";
+  version = "11";
 
   src = fetchFromGitHub {
-    owner = "franglais125";
+    owner = "poehlerj";
     repo = "no-title-bar";
-    rev = "v${version}";
-    sha256 = "02zm61fg40r005fn2rvgrbsz2hbcsmp2hkhyilqbmpilw35y0nbq";
+    rev = "V_11";
```

Comment:
```suggestion
    rev = "V_${version}";
```

Reply:
Fixed. Thanks for catching that, despite doing so in my personal repo somehow I didn't here? Oops :sweat_smile: